PurchasesCoreSwift/FoundationExtensions/String+Extensions.swift

```diff
@@ -22,6 +22,7 @@ extension String {
 
 }
 
+// swiftlint:disable identifier_name
```

Comment:
Exist a `var` called `i`, and due to that, it's breaking this rule. 

In fact, there are other parts of the code that break this rule as well. I found these:
- `Data+Extensions.swift:35`
- `Data+Extensions.swift:40`
- `SwiftStyleGuide.swift:140`

These errors will be fixed when this rule will be enabled again, I guess

Reply:
I'm working on enabling swiftlint right now, so these will be addressed very soon!